### PR TITLE
test(gateway configuration): fix existing tests (TECH-468)

### PR DESCRIPTION
**Pull request by Mohammer5**
Fixes TECH-468

Existing tests are now running again

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 const plugins = require('@dhis2/cli-utils-cypress/plugins')
 
 module.exports = (on, config) => {
```
Comment (ismay) [inline cypress/plugins/index.js:3]: Interesting, so this is how we can solve the new cookie policy?

File: cypress/support/index.js [hunk 2]
```diff
@@ -1,8 +1,68 @@
...
+            },
+        })
+    })
+
+    beforeEach(() => {
+        // This ensures the app platform knows which URL to use even if REACT_APP_DHIS2_BASE_URL is undefined
+        // It also ensures that the value from the cypress env is used instead of REACT_APP_DHIS2_BASE_URL
+        const baseUrl = getApiBaseUrl()
+        localStorage.setItem('DHIS2_BASE_URL', baseUrl)
+        console.log('localStorage', localStorage)
```
Comment (ismay) [inline cypress/support/index.js:64]: Is this console.log still meant to be in here? (Interesting to read this btw, this module I mean, seems quite straightforward)

**Review by ismay — APPROVED** [review]
Looks good 👍!

File: cypress/plugins/index.js (hunk 1 above)
Comment (Mohammer5) [reply cypress/plugins/index.js:3]: yes, I'll try to put that in to the cypress repo and replace it here

File: cypress/support/index.js (hunk 2 above)
Comment (Mohammer5) [reply cypress/support/index.js:64]: Oh, no, will remove that one. Same like the plugin one, I'll try to move that one to the cypress repo so we can use it in all apps (originally written by @HendrikThePendric, I just extracted the relevant parts)

**Comment by ismay** [comment]
Ah wait, the changes look good, but it didn't actually run cypress on CI? Isn't that something we do for PRs with this app?

**Review by ismay — CHANGES_REQUESTED** [review]
I assumed cypress tests were run on CI, but I can't find them there. When I went to test them locally, I noticed that the package.json scripts are using the old scripts name (cypress:start, should be cy:start):

```
  12   │         "cy:start": "BROWSER=none yarn start",
  13   │         "cy:open": "d2-utils-cypress open --appStart 'yarn cypress:start'",
  14   │         "cy:run": "d2-utils-cypress run --appStart 'yarn cypress:start'",
```

**Comment by Mohammer5** [comment]
Good catch. They're not meant to be run automatically yet. I'll fix the broken scripts though

**Comment by Mohammer5** [comment]
@ismay done

**Review by ismay — APPROVED** [review]
Tested it locally and all tests pass, looks good! 👍

**Pull request merged**